be/src/exec/tablet_sink.cpp

```diff
@@ -845,40 +845,35 @@ Status OlapTableSink::close(RuntimeState* state, Status close_status) {
 }
 
 void OlapTableSink::_print_varchar_error_msg(RuntimeState* state, const Slice& str, SlotDescriptor* desc) {
-    std::stringstream ss;
-    ss << "the length of input is too long than schema. "
-       << "column_name: " << desc->col_name() << "; "
-       << "input_str: [" << str.to_string() << "] "
-       << "schema length: " << desc->type().len << "; "
```

Comment:
For the length considered by the system, the actual length of string may not be omitted.
because 
1. It is unhappy for users to count how long the string is.
2. The user may not know that Chinese characters occupy 3 lengths at all, so that user could not find the problem.